### DISCUSSION TITLE
WS2812 as status LED

### DISCUSSION THREAD

**Discussion by tim-hilt** [arendst/Tasmota#22368] [Q&A] [answered]
Can I configure a WS2812 led as a status led using commands? What’s the recommended way of doing this?

**Comment by sfromis** [comment, marked as answer]
No, Tasmota does not support such a configuration. You'd have to do something else to mimic the status led function, like reacting to relevant events, and do what you'd like there.

Status leds can only be simple leds, possibly controlled as PWM for variable brightness.

**Reply by tim-hilt** [reply]
Thanks! That helps.

**Reply by tim-hilt** [reply]
Would something like this work?

```
Backlog Rule1 ON System#Boot DO Color #FF0000 ENDON ON Wifi#Connected DO IF Mqtt#Connected==0 THEN Color #0000FF ENDIF ENDON ON Mqtt#Connected DO Color #00FF00 ENDON ON Wifi#Disconnected DO Color #FF0000 ENDON ON Mqtt#Disconnected DO IF Wifi#Connected==1 THEN Color #0000FF ENDIF ENDON; Rule1 1
```

Didn't come up with this myself though, not that experienced with rules yet.

**Reply by sfromis** [reply]
No, that's not how rules work. Using ChatGPT can only be recommended if you want to be confused, or are able to fix the expected errors yourself.

**Reply by tim-hilt** [reply]
Alright, thank you. I‘ll do some digging myself then.
